cmd/descheduler/app/server.go

```diff
@@ -57,5 +61,20 @@ func NewDeschedulerCommand(out io.Writer) *cobra.Command {
 }
 
 func Run(rs *options.DeschedulerServer) error {
+	if rs.EnableProfiling {
+		timeNow := time.Now().Format("20060102150405")
+		// As of now, adding only CPU & memory profiles, we can include
+		// more profiles like blocking, trace, mutex. When we move to http
+		// based profiling all those profiles are automatically added.
+		cpuprofile, err := os.Create("cpu" + timeNow + ".pprof")
+		// The pprof file is of format cpu20171130152506.pprof
+		heapprofile, err := os.Create("heap" + timeNow + ".pprof")
+		if err != nil {
+			glog.Errorf("%v", err)
+		}
+		pprof.StartCPUProfile(cpuprofile)
```

Comment:
can we create these files when running in a pod?

Reply:
I didn't test in a pod, were you getting an error when running descheduler in a pod?

Reply:
yes permission denied error.